src/ui/PrintableUri/PrintableUri.ts

```diff
@@ -150,12 +151,23 @@ export class PrintableUri extends Component {
     });
     const link = new ResultLink(this.buildElementForResultLink(this.result.printableUri), this.options, this.bindings, resultPart);
     this.links.push(link);
-    this.element.appendChild(link.element);
+    this.element.appendChild(this.makeLinkAccessible(link.element));
+  }
+
+  private makeLinkAccessible(link: HTMLElement) {
+    return $$(
+      'span',
+      {
+        className: 'coveo-printable-uri-part',
```

Comment:
What was the reason for grouping the class name with making the link accessible vs. when building the element?

The method is behaving more as `makeLinkPrintableUriAndAccessible`

Reply:
This was done to make sure that the top-level element is the one with the `coveo-printable-uri-part` class name. However, for the purpose of readability, it would be possible to modify `makeLinkAccessible` to remove `coveo-printable-uri-part` from `link` and have the anchor (`a`) element have `coveo-printable-uri-part` when it is initially created in `buildElementForResultLink`.